ui/v2.5/src/components/Performers/PerformerDetails/Performer.tsx

```diff
@@ -592,16 +591,32 @@ const PerformerPage: React.FC<IProps> = ({ performer }) => {
   );
 };
 
-const PerformerLoader: React.FC = () => {
-  const { id } = useParams<{ id?: string }>();
-  const { data, loading, error } = useFindPerformer(id ?? "");
+const PerformerLoader: React.FC<RouteComponentProps<IPerformerParams>> = ({
+  location,
+  match,
+}) => {
+  const { id, tab } = match.params;
+  const { data, loading, error } = useFindPerformer(id);
+
+  useScrollToTopOnMount();
 
   if (loading) return <LoadingIndicator />;
   if (error) return <ErrorMessage error={error.message} />;
   if (!data?.findPerformer)
     return <ErrorMessage error={`No performer found with id ${id}.`} />;
 
-  return <PerformerPage performer={data.findPerformer} />;
+  if (!tab || !isTabKey(tab)) {
+    return (
+      <Redirect
+        to={{
+          ...location,
+          pathname: `/performers/${id}/${defaultTab}`,
```

Comment:
Yeah I don't use any userscripts so I hadn't thought of that. I do think that the change is useful though, because I would imagine that for example the performer page opening on the scenes tab is annoying for anyone who uses stash primarily for images.

I think I will revert that change for the sake of this PR though, since it is the only "breaking" change.

Reply:
Related https://github.com/stashapp/stash/issues/3961

Reply:
I've pushed a change, the behaviour should now match how it was before.
I hadn't actually seen that issue, after this merges I'll make a PR to add that functionality - it should be straightforward to get working.